tests/test_python.py

```diff
@@ -19,6 +20,7 @@
 # copy tensorflow module trainer from the local application directory to the container
 DOCKERF_PY_T1 = f"""
 WORKDIR {bcdir}
+COPY {orig + appdir}/{appl1}  {bcdir}
```

Comment:
Why is this change necessary?

Reply:
Used in a previous debugging, now no more needed,it will be removed